libraries/commerce/product/constants/index.js

```diff
@@ -70,3 +70,6 @@ export const PRODUCT_RELATIONS_TYPE_UPSELLING = 'upselling';
 export const PRODUCT_RELATIONS_TYPE_BONUS = 'bonus';
 export const PRODUCT_RELATIONS_TYPE_BOUGHT_WITH = 'boughtWith';
 export const PRODUCT_RELATIONS_TYPE_CUSTOM = 'custom';
+
+// UPDATE METADATA
```

Comment:
Redundant, please remove

Reply:
we have these comments everywhere. i changed to comment